src/sdk/pynni/nni/parameter_expressions.py

```diff
@@ -32,12 +32,14 @@ def choice(options, random_state):
     return random_state.choice(options)
 
 
-def randint(upper, random_state):
+def randint(lower, upper, random_state):
     '''
+    Generate a random integer from `low` (inclusive) to `high` (exclusive).
```

Comment:
low/high or lower/upper?

Reply:
I should've changed all `upper` to `high`, to conform with naming conventions of parameters in tuners. If there is still one I've missed, plz let me know.

Reply:
lower/upper.

Reply:
@ultmaster please update to `lower` and `upper`